pkg/xen-tools/Dockerfile

```diff
@@ -26,10 +26,12 @@ ENV BUILD_PKGS \
     gettext iasl util-linux-dev ncurses-dev glib-dev \
     pixman-dev libaio-dev yajl-dev argp-standalone \
     linux-headers git patch texinfo tar libcap-ng-dev \
-    attr-dev flex bison cmake libusb-dev zstd-dev
+    attr-dev flex bison cmake libusb-dev zstd-dev \
+    python3 build-base ninja
+
 ENV BUILD_PKGS_arm64 dtc-dev
 
-ENV PKGS alpine-baselayout musl-utils bash libaio libbz2 glib pixman yajl keyutils libusb xz-libs libuuid sudo zstd-libs
+ENV PKGS alpine-baselayout libgcc musl-utils bash libaio libbz2 glib pixman yajl keyutils libusb xz-libs libuuid sudo zstd-libs
```

Comment:
@roja-zededa , you didn't answer @shjala 's comment in the previous versions of this PR... why do you need this package? Apparently it builds without it, libgcc will add only a few KBs, but I'm wondering why is needed it....

Reply:
Plus `sudo` is famous for being buggy, since we are moving towards non-root containers, I suggest removing it if everything works without it.